lib/kafka/datadog.rb

```diff
@@ -108,6 +108,18 @@ def emit(type, *args, tags: {})
       end
     end
 
+    class CompressionSubscriber < StatsdSubscriber
+      def compress(event)
+        message_count = event.payload.fetch(:message_count)
+        compressed_bytesize = event.payload.fetch(:compressed_bytesize)
+
+        gauge("compressor.compressed_bytesize", compressed_bytesize)
+        count("compressor.message_count", message_count)
+      end
```

Comment:
I think we should add another histogram here – the compression _factor_. That's much easier to use in comparisons and evaluations of efficacy.

```ruby
histogram("compressor.compression_factor", uncompressed_bytesize / compressed_bytesize)
```